elastic/kibana
# [Fleet] Reload settings page when clicking on hosts flyout link

## Summary
Closes https://github.com/elastic/kibana/issues/145547 and https://github.com/elastic/kibana/issues/144965

Reload settings page when clicking on "Settings" link in fleet server hosts flyout link and makes sure that the flyout is closed.

### Test steps
- From settings page, add a new fleet server host
- After the host is added, click on "settings" link. The settings page should reload and show the new host right away.
This fix is applied to both "Quick start" and "Advanced" tab

<img width="1056" alt="Screenshot 2023-01-10 at 11 44 58" src="https://user-images.githubusercontent.com/16084106/211531274-0907a63b-2a39-47e2-bf03-04926d945237.png">
